resources/assets/less/variables.less

```diff
@@ -22,6 +22,7 @@
 @font-default: 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Kaku Gothic ProN', Meiryo, 'Microsoft YaHei', 'Apple SD Gothic Neo', sans-serif;
 @font-default-vi: Exo, 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Kaku Gothic ProN', Meiryo, 'Microsoft YaHei', 'Apple SD Gothic Neo', sans-serif;
 @font-default-zh: 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Sans GB', 'Microsoft YaHei', 'Apple SD Gothic Neo', system-ui, sans-serif;
+@font-default-zh-tw: 'Exo 2', 'Helvetica Neue', Tahoma, Arial, 'Hiragino Sans TC', 'Microsoft JhengHei', 'Apple SD Gothic Neo', system-ui, sans-serif;
```

Comment:
`Hiragino Sans GB` is a MacOS font, `Hiragino Sans TC` isn't

Reply:
Ah! I didn't realize it was bundled in macOS. Then, I'll change it to PingFang TC.